packages/@storefront/core/src/core/tag/mixins/provide-consume.ts

```diff
@@ -34,10 +34,11 @@ export namespace ProvideConsume {
           this.one(Phase.UNMOUNT, () => {
             aliasTag.off(Phase.UPDATE, resetUpdated);
             aliasTag.off(Phase.UPDATED, softUpdateDependant);
+            // allow alias tag to be garbage-collected
+            aliasTag = null;
```

Comment:
@jmbtutor

To confirm: since the `aliasTag` object (and it's associated properties) will be garbage collected, the thinking here is that we don't need to explicit remove the `markUpdated` listener?

Reply:
No, it's that Riot unregisters all of its listeners for a given component when that component unmounts, so doing it ourselves is redundant.

Reply:
Noted, thanks for clarifying.